project/5000user5000/README.rst

```diff
@@ -0,0 +1,87 @@
+Efficient Mixed-Precision Inference Engine
+=========================================
+
+
+Basic Information
+=================
+
+- **Project**: mpGEMM
+- **Github**: https://github.com/5000user5000/mpGEMM
+
+On resource-constrained devices, such as embedded systems, running deep learning models, especially LLMs, is highly computationally expensive. Low-bit quantization is a popular solution to reduce memory consumption. However, when weights are quantized below 8 bits, performing matrix multiplication between the weight matrix and an FP16 activation matrix requires dequantization to a common precision, as hardware lacks native support for mixed-precision matrix multiplication (mpGEMM). Some recent research suggests using lookup tables (LUTs) to replace dequantization, further reducing computational overhead.
+
+.. image:: ./img/lut.png
+
+
+Problem to Solve
+================
+
+- Develop a **low-level numerical computation core** in C++ for high-speed matrix operations.
```

Comment:
You won't have a lot to present if your implementation is slower than vendor-provided libraries (MKL, Accelerate, etc.), but it is very hard to beat the performance of vendor library.

Do you have an idea how much faster vendor library is compared to naive implementation?